src/components/CalendarComponents/categoriesBar.vue

```diff
@@ -25,24 +25,14 @@
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator';
+import { Component, Vue, Prop } from 'vue-property-decorator';
+
 import Category from '@/models/Category';
-import { AllCategories } from '@/db-service/Categories/queries';
 
-@Component({
-  apollo: {
-    allCategories: {
-      query: AllCategories,
-    },
-  },
-})
+@Component
 export default class CategoriesBar extends Vue {
-  get categories(): Category[] {
```

Comment:
Why did you remove the categories from the component?

Reply:
Answered below...